tests/zfs-tests/tests/functional/cli_root/zfs_destroy/cleanup.ksh

```diff
@@ -27,6 +27,6 @@
 
 . $STF_SUITE/tests/functional/cli_root/zfs_destroy/zfs_destroy_common.kshlib
 
-cleanup_testenv
+#cleanup_testenv
```

Comment:
@behlendorf 
zfs_destroy test suite, Each individual destroy test case, All have a cleanup_testenv when the test case is log_onexit ,  For example log_onexit cleanup_testenv,  Therefore, zfs_destroy cleanup.ksh  again execute cleanup_testenv is unnecessary.